src/test/resources/com/puppycrawl/tools/checkstyle/grammar/javadoc/expectedJavadocWithCrAsNewlineAst.txt

```diff
@@ -1,20 +1,20 @@
 JAVADOC -> JAVADOC [0:0]
 |--NEWLINE -> \r [0:0]
-|--LEADING_ASTERISK -> * [0:1]
-|--TEXT ->  Checks that commentContent starts with '*' javadoc comment identifier. [0:2]
-|--NEWLINE -> \r [0:73]
-|--LEADING_ASTERISK ->     * [0:74]
-|--TEXT -> &lt;property name="ignorePattern" value="^ *\* *[^ ]+$"/&gt* [0:79]
-|--NEWLINE -> \r [0:139]
-|--LEADING_ASTERISK -> \t* [0:140]
-|--TEXT ->  This * is not a leading * [0:142]
-|--NEWLINE -> \r [0:168]
-|--LEADING_ASTERISK -> * [0:169]
-|--WS ->   [0:170]
-|--JAVADOC_TAG -> JAVADOC_TAG [0:171]
-|   |--RETURN_LITERAL -> @return [0:171]
-|   |--WS ->   [0:178]
-|   `--DESCRIPTION -> DESCRIPTION [0:179]
-|       |--TEXT -> true if commentContent starts with '*' javadoc comment identifier [0:179]
-|       `--NEWLINE -> \r [0:244]
-`--EOF -> <EOF> [0:245]
+|--LEADING_ASTERISK -> * [1:0]
+|--TEXT ->  Checks that commentContent starts with '*' javadoc comment identifier. [1:1]
+|--NEWLINE -> \r [1:72]
+|--LEADING_ASTERISK ->     * [2:0]
+|--TEXT -> &lt;property name="ignorePattern" value="^ *\* *[^ ]+$"/&gt* [2:5]
+|--NEWLINE -> \r [2:65]
+|--LEADING_ASTERISK -> \t* [3:0]
+|--TEXT ->  This * is not a leading * [3:2]
+|--NEWLINE -> \r [3:28]
+|--LEADING_ASTERISK -> * [4:0]
+|--WS ->   [4:1]
+|--JAVADOC_TAG -> JAVADOC_TAG [4:2]
+|   |--RETURN_LITERAL -> @return [4:2]
+|   |--WS ->   [4:9]
+|   `--DESCRIPTION -> DESCRIPTION [4:10]
+|       |--TEXT -> true if commentContent starts with '*' javadoc comment identifier [4:10]
+|       `--NEWLINE -> \r [4:75]
+`--EOF -> <EOF> [5:0]
```

Comment:
This was the test from https://github.com/checkstyle/checkstyle/issues/2329, but now it shows the correct line numbers.